docs/content/dagster-plus/managing-deployments/alerts.mdx

```diff
@@ -79,10 +79,15 @@ Alert policies are configured on a **per-deployment basis**. This means, for exa
         <strong>Run alert</strong>
       </td>
       <td>
-        Triggers on job run success or failure; may optionally include a set of
-        configured tags. If an alert policy has no configured tags, all jobs
-        will be eligible for that alert. Otherwise, only jobs that contain all
-        the tags for a given alert policy are eligible for that alert.
+        Triggers on job run success, failure, or time limit exceeded; may
+        optionally include a set of configured tags. If an alert policy has no
+        configured tags, all jobs will be eligible for that alert. Otherwise,
+        only jobs that contain all the tags for a given alert policy are
+        eligible for that alert.
+        <br />
+        Each newly created organization starts with a long-running run alert
+        policy, directed at the organization owner's email address, which will
```

Comment:
> directed at the organization owner's email address, which will

Does this mean the user in the org who has the **Organization Admin** role?

Reply:
There can be multiple organization admins, so by organization owner I hoped to disambiguate that from the person who initially created the organization (that's who is getting the alert).

Reply:
Is there a way to identify this user in the UI? I quickly poked around a bit but didn't see anything obvious

Reply:
There's no way to identify this user in the UI. It's just the person who goes through the onboarding flow initially.

Reply:
Gotcha. I think we should consider adding a label or something for this in the UI. At a previous job, we had something similar for the app's billing contact and it lead to a ton of support questions and confusion.

Reply:
That's a very good idea; I'll make a linear ticket for it (IMO we shouldn't block this PR on that though)

Reply:
Gonna change it to just: the email address you originally created the organization with.

Reply:
Sounds good to me!